config/dynamicconfig/development.yaml

```diff
@@ -30,4 +30,6 @@ history.defaultWorkflowRetryPolicy:
     MaximumAttempts: 0
 system.advancedVisibilityWritingMode:
   - value: "off"
-    constraints: {}
\ No newline at end of file
+    constraints: {}
+worker.executionsScannerEnabled:
+  - value: true
```

Comment:
This doesn't look like "disable by default". If you decide to leave it here, please add it to "development_es.yaml" also.